src/lib/source-handlers/azure/list-repos.ts

```diff
@@ -59,7 +60,17 @@ async function getRepos(
     ),
   );
   if (data.statusCode != 200) {
-    new Error(`Failed to fetch page: ${url}\n${data.body}`);
+    if (data.statusCode === 429) {
```

Comment:
drop this whole block and call you new function above instead

Reply:
Done